app/src/chainlit.py

```diff
@@ -260,14 +260,25 @@ async def _batch_proccessing(file: AskFileResponse) -> None:
         # E.g., "abcd.csv" to "abcd_results.csv"
         result_file_name = file.name.removesuffix(".csv") + "_results.csv"
 
+        elements = [cl.File(name=result_file_name, path=result_file_path)]
         await cl.Message(
+            author="backend",
             content="File processed, results attached.",
-            elements=[cl.File(name=result_file_name, path=result_file_path)],
+            metadata={
+                "status": "processing_complete",
+                "original_file": file.name,
+                "result_file": result_file_name
+            },
+            elements=elements
         ).send()
 
     except ValueError as err:
         await cl.Message(
             author="backend",
-            metadata={"error_class": err.__class__.__name__, "error": str(err)},
-            content=f"{err.__class__.__name__}: {err}",
+            content=f"Error processing file: {err}",
+            metadata={
+                "status": "processing_error",
```

Comment:
`processing_error` sounds too generic to be useful. How about:
```suggestion
                "status": "batch_processing_error",
```

Reply:
Resolving as I removed the metadata key from Message objects